globals.go

```diff
@@ -369,9 +369,11 @@ const (
 	// indicating that there are still running tasks.
 	PRTasksRunningDescription = "tasks are running"
 
-	// HostServicePasswordExpansion is the expansion for the service password that is stored on the host,
-	// and is meant to be set as a private variable so that it will be redacted in all logs.
-	HostServicePasswordExpansion = "host_service_password"
+	// HostServicePasswordInternalRedaction is the service user's password on a
+	// host. This is for Evergreen internal use only, and while it should not
+	// typically be accessible, it should be redacted in any logs from the task
+	// if it does happen to appear.
+	HostServicePasswordInternalRedaction = "host_service_password"
```

Comment:
Is HostServicePasswordExpansion still a more clear name, since this is the expansion and not the password itself?

Reply:
(also since this is only used by rest/route, we can just define it in that package as a local constant rather than a global constant)

Reply:
This isn't really an expansion at all, because it functionally isn't usable like an expansioni s. This is just a placeholder name used for redaction if the password somehow happens to appear in plaintext in the logs.

Renamed to `hostServicePasswordPlaceholder` and moved into the relevant package.